### ISSUE TITLE
Support conditions?

### ISSUE THREAD

**Issue by jprichardson** [jprichardson/node-suppose#7]
http://stackoverflow.com/questions/21462939/how-can-i-use-conditions-properly-with-node-suppose

**Comment by stevenvachon** [comment]
Is this supported via the callback function in `respond()`?

**Comment by piranna** [comment]
> Is this supported via the callback function in respond()?

Yes, functions on `respond()` can be used for this, and probably it's the best option.
